build: fix path of @angular/ssr package

alan-agius4 opened the pull request.

alan-agius4 committed:
build: fix path of `@angular/ssr` package

The path was incorrect

devversion commented on .gitignore:
```diff
@@ -4,7 +4,8 @@
 *.log
-/node_modules/
+node_modules/
+**/node_modules/
```
I intentionally removed such pattern so that we don't accidentally have nested `node_modules/` in the repository that could then slow-down Bazel actions by e.g. 20min. This then motivates adding to `bazelignore` + `gitignore` explicitly.

alan-agius4 committed:
build: correct ignore nested node_modules

The commit updates both Git and Bazel ignore config to ingore nested node_modules

alan-agius4 committed:
build: remove unused patch

This commit removes an unused patch

devversion reviewed (approved):
Ty for the cleanup!

alan-agius4 committed:
fixup! build: correct ignore nested node_modules

thePunderWoman commented:
This PR was merged into the repository by commit 7fd034e13fe8b3778991d5cee4971b1d49daab90.

The changes were merged into the following branches: main, 19.0.x
